conversion/binary_to_octal.py

```diff
@@ -0,0 +1,29 @@
+def binary_to_octal(msg: str) -> str:
```

Comment:
```suggestion
def binary_to_octal(binary: str) -> str:
```
ganti ke binary untuk lebih mudah pembacaan bukan ``msg``